tests/src/TestFixtureBase.cpp

```diff
@@ -4,13 +4,11 @@
 #include "TestResult.h"
 #include "braincloud/AuthenticationType.h"
 
-#ifdef _WIN32
-#define getcwd _getcwd // stupid MSFT "deprecation" warning
-#endif
-
 using namespace std;
 
+#ifndef ID_FILE_NAME
```

Comment:
This will allow me to add the path in it for the PS4 and other platforms with specific working dir by defining the preprocessor in project without having to modify this source.